firstvoices/backend/tests/test_apis/base_api_test.py

```diff
@@ -654,11 +654,15 @@ def test_update_instance_missing_404(self):
         assert response.status_code == 404
 
     @pytest.mark.django_db
-    def test_update_success_200(self):
+    def perform_success_request(self, data_with_nulls=False):
```

Comment:
Could we name this perform_detail_request instead? Nothing guarantees it will succeed ;)

Reply:
that is correct. I will add this change.